From bc664cd54b990135947cf73b5728fce80de781b3 Mon Sep 17 00:00:00 2001
From: Victor Zverovich <viz@meta.com>
Date: Tue, 28 May 2024 08:57:01 -0700
Subject: [PATCH] Fix CMake config

Summary: `ast/visitor.cc` has been removed, update the CMake config accordingly.

Reviewed By: yoney

Differential Revision: D57861871

fbshipit-source-id: 3da25918196eea55daf430cc333fd1843ab43402
---
 thrift/compiler/CMakeLists.txt | 1 -
 1 file changed, 1 deletion(-)

diff --git a/thrift/compiler/CMakeLists.txt b/thrift/compiler/CMakeLists.txt
index 7b483929f05..837f7169e7e 100644
--- a/thrift/compiler/CMakeLists.txt
+++ b/thrift/compiler/CMakeLists.txt
@@ -73,7 +73,6 @@ add_library(
   ast/t_structured.cc
   ast/t_type.cc
   ast/t_typedef.cc
-  ast/visitor.cc
 )
 target_link_libraries(
   compiler_ast
